girder/utility/s3_assetstore_adapter.py

```diff
@@ -165,12 +165,12 @@ def __init__(self, assetstore):
         """
         :param assetstore: The assetstore to act on.
         """
+        super(S3AssetstoreAdapter, self).__init__(assetstore)
         if ('accessKeyId' in assetstore and 'secret' in assetstore and
                 'service' in assetstore):
             assetstore['botoConnect'] = makeBotoConnectParams(
                 assetstore['accessKeyId'], assetstore['secret'],
                 assetstore['service'])
-        self.assetstore = assetstore
```

Comment:
It doesn't matter currently, but would it be more appropriate to call `super` after modifying the document for forward compatibility?